Realm/Realm/Linq/ChangeSet.cs

```diff
@@ -61,13 +63,21 @@ public class ChangeSet
         /// <value>An array of <see cref="Move"/> structs, indicating the source and the destination index of the moved row.</value>
         public Move[] Moves { get; }
 
-        internal ChangeSet(int[] insertedIndices, int[] modifiedIndices, int[] newModifiedIndices, int[] deletedIndices, Move[] moves)
+        /// <summary>
+        /// Gets a value indicating whether the collection has been cleared.
+        /// The collection is cleared after calling methods such as <see cref="IList.Clear"/>.
+        /// </summary>
+        /// <value><c>true</c> if the collection has been cleared; <c>false</c> otherwise.</value>
+        public bool Cleared { get; }
```

Comment:
```suggestion
        public bool IsCleared { get; }
```
I suggest the `Is<Adjective>` form to better match the rest of .NET.